contracts/external/traders/CanonicalOrders.sol

```diff
@@ -178,8 +178,8 @@ contract CanonicalOrders is
         bytes32 indexed orderHash,
         address indexed orderMaker,
         uint256 fillAmount,
-        uint256 totalFilledAmount,
```

Comment:
Why are we removing this, is it not needed?

Reply:
we're not using it, no